concent_api/generate-requirements-lock.sh

```diff
@@ -1,10 +1,25 @@
 #!/bin/bash -e
 
+file_name="$1"
+
 if [[ "$VIRTUAL_ENV" != "" ]]
 then
-    echo "--find-links https://builds.golem.network/simple/pyelliptic" > ${BASH_SOURCE%/*}/requirements.lock
-    pip freeze >> ${BASH_SOURCE%/*}/requirements.lock
-    echo "Done"
+    if [ "$#" -ne 1 ]
+    then
+        echo "Script should be called with exactly one parameter"
+        exit 1
+    else
+        if [ -s "$file_name" ] && [[ "$file_name" == *.lock ]]
+        then
+            additional_lines=$(cat $file_name | grep "^--")
```

Comment:
I would stay with `grep "^--"` because if we want to use this script with requirements files which would have different lines than `--find-links` it won't work.

Reply:
Copying all those lines blindly is not a good idea. What if the file contains this:

``` bash
--editable git://github.com/golemfactory/golem-messages.git@v2.12.0#egg=golem_messages
```

Reply:
Ok, so changed to version:
```
additional_lines=$(cat $file_name | grep "^--find-links")
```